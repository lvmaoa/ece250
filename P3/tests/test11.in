i goat
i cat
i dog
i picture
i frame
i pattern
i chased
p
exit
