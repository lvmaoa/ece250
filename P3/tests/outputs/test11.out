success
success
success
success
success
success
success
cat chased dog frame goat pattern picture 
